deploy/templates/bicep/ContainerAppSetup.bicep

```diff
@@ -0,0 +1,385 @@
+        {
+          image: '${containerRegistry.name}.azurecr.io/${fhirTransformationImage}:${imageTag}'
+          name: 'fhir-transformation-${timestamp}'
+          env: [
+            {
+              name: 'APPINSIGHTS_INSTRUMENTATIONKEY'
+              value: appInsights.properties.InstrumentationKey
+            }
+            {
+              name: 'EventBatching__FlushTimespan'
+              value: '300'
+            }
+            {
+              name: 'EventBatching__MaxEvents'
+              value: '300'
+            }
+            {
+              name: 'Checkpoint__BlobPrefix'
+              value: 'MeasurementToFhir'
+            }
+            {
+              name: 'CheckpointStorage__AuthenticationType'
+              value: 'ManagedIdentity'
+            }
+            {
+              name: 'CheckpointStorage__BlobStorageContainerUri'
+              value: '${storageAccount.properties.primaryEndpoints.blob}checkpoint'
+            }
+            {
+              name: 'CheckpointStorage__BlobContainerName'
+              value: 'checkpoint'
+            }
+            {
+              name: 'TemplateStorage__AuthenticationType'
+              value: 'ManagedIdentity'
+            }
+            {
+              name: 'TemplateStorage__BlobStorageContainerUri'
+              value: '${storageAccount.properties.primaryEndpoints.blob}template'
+            }
+            {
+              name: 'TemplateStorage__BlobContainerName'
+              value: 'template'
+            }
+            {
+              name: 'FhirClient__UseManagedIdentity'
+              value: 'true'
+            }
+            {
+              name: 'FhirService__Url'
+              value: fhirService.properties.authenticationConfiguration.audience
+            }
+            {
+              name: 'InputEventHub__AuthenticationType'
+              value: 'ManagedIdentity'
+            }
+            {
+              name: 'InputEventHub__EventHubNamespaceFQDN'
+              value: eventhubNamespace.properties.serviceBusEndpoint
+            }
+            {
+              name: 'InputEventHub__EventHubConsumerGroup'
+              value: '$Default'
+            }
+            {
+              name: 'InputEventHub__EventHubName'
+              value: 'devicedata'
+            }
+            {
+              name: 'NormalizationEventHub__AuthenticationType'
+              value: 'ManagedIdentity'
+            }
+            {
+              name: 'NormalizationEventHub__EventHubNamespaceFQDN'
+              value: eventhubNamespace.properties.serviceBusEndpoint
+            } 
+            {
+              name: 'NormalizationEventHub__EventHubConsumerGroup'
+              value: '$Default'
+            }
+            {
+              name: 'NormalizationEventHub__EventHubName'
+              value: 'normalizeddata'
+            }
+            {
+              name: 'ResourceIdentity__ResourceIdentityServiceType'
+              value: 'Create'
```

Comment:
Ideally this should be a parameter the customer can define to switch between Create and Lookup. The tags will also need to be updated based on this parameter i.e. IomtFhirConnector: 'ResourceIdentity:Create' would IomtFhirConnector: 'ResourceIdentity:Lookup' if Lookup is selected.

Reply:
Got it, I will add that in.